packages/keyring-controller/CHANGELOG.md

```diff
@@ -6,14 +6,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.1.0]
+### Changed
+- add rollbackToPreviousProvider method ([#1132](https://github.com/MetaMask/core/pull/1132))
```

Comment:
This change didn't really impact this package at all; it only affected the unit tests.

Reply:
This package isn't affected by the other packages changes either (i.e. it's not dependent on any packages with a breaking change).

So... maybe we can omit this package from this release? Or include it as a patch, with an entry just saying that we've updated the minimum supported versions of the other core packages (this controller references 4 others)